twilio/twiml/voice_response.py

```diff
@@ -630,7 +631,20 @@ def ssml_break(self, strength=None, time=None, **kwargs):
         """
         return self.nest(SsmlBreak(strength=strength, time=time, **kwargs))
 
-    def ssml_emphasis(self, words, level=None, **kwargs):
+    @deprecated_method(break_)
+    def ssml_break(self, strength=None, time=None, **kwargs):
```

Comment:
Is the arg for the decorator a string or a method?

Reply:
Oops